src/SSHDebugPS/ConnectionManager.cs

```diff
@@ -149,5 +174,140 @@ public static Connection GetSSHConnection(string name)
 
             return null;
         }
+
+        public static IEnumerable<ConnectionInfo> GetAvailableSSHConnectionInfos()
+        {
+            ConnectionInfoStore store = new ConnectionInfoStore();
+
+            return store.Connections.ToList().Select(item => (ConnectionInfo)item);
+        }
+
+        private const string dockerCommand = "docker";
+        // --no-trunc avoids parameter truncation
+        private const string dockerPSArgs = "ps --no-trunc --format \"{{json .}}\"";
+        public static IEnumerable<IContainerInstance> GetLocalDockerContainers()
```

Comment:
>GetLocalDockerContainers [](start = 54, length = 24)

Are you planning to someday better unify the local and remote cases? They are sure very similar.

Reply:
Yes. I ran into a timing issue on the local case which forced me to create a .Run() method on the local and then i had to handle error output slightly different. 

My plan is when I go back and clean up the CommandRunner and the ShellExecutionManager to work that out too.